tests/rules/property.js

```diff
@@ -111,7 +111,19 @@ eslintTester.run("property", rule, {
         {
             code: "w.innerHTML = `<span>${'lulz'+'meh'}</span>`;",
             parserOptions: { ecmaVersion: 6 }
-        }
+        },
+
+        // testing unwrapSafeHTML spread
+        {
+            code: "this.imeList.innerHTML = Sanitizer.unwrapSafeHTML(...listHtml);",
+            parserOptions: { ecmaVersion: 6 }
+        },
+
+
+        // Native method (Check customize code doesn't include these)
+        {
+            code: "document.toString = evil;"
```

Comment:
oh gotcha. Overwriting the built-ins will screw us up badly.
I'm OK with this, but I want to remind you that we are trying to prevent accidents, not maliciousness.